src/openklant/components/klantinteracties/api/tests/test_filters.py

```diff
@@ -40,16 +40,31 @@ def setUp(self):
             self.klantcontact4,
             self.klantcontact5,
         ) = KlantcontactFactory.create_batch(5)
-        for betrokkene_klantcontact in [
-            self.klantcontact,
-            self.klantcontact2,
-            self.klantcontact3,
-            self.klantcontact4,
-            self.klantcontact5,
-        ]:
-            self.betrokkene = BetrokkeneFactory.create(
-                klantcontact=betrokkene_klantcontact
-            )
+
+        self.partij, self.partij2, self.partij3, self.partij4, self.partij5 = (
```

Comment:
I'm not really a fan of setting up test data like this in the `setUp` function. In order to grasp each test you will have to lookup the test data created in this function. Also, whenever tests increase or their requirements change, this function might become bigger and bigger while not all tests use the data that is created here.

Creating test data in the tests themself is in my opinion more explicit and prevents having to deal with extra data which might not be needed for a test (when `setUp` data is used, as that will affect all tests). This will also decrease the time it will take to run tests as only relevant data would have to be created.

Reply:
Me neither. I already wanted to change this but thought that this would be better to do outside of this pr for development speed.